init.lua

```diff
@@ -59,19 +59,34 @@ end
 
 -- Usage:
 --   resizer = SkyRocket:new({
+--     opacity = 0.3,
 --     moveModifiers = {'cmd', 'shift'},
+--     moveMouseButton = 'left',
 --     resizeModifiers = {'ctrl', 'shift'}
+--     resizeMouseButton = 'left',
 --   })
 --
+local function buttonNameToEventType(name)
+    if name == 'left' then
```

Comment:
**chore** Please match the local indentation style in this file (2 spaces), thanks!